.rubocop.yml

```diff
@@ -59,3 +59,6 @@ Style/Documentation:
   Exclude:
     - 'lib/**/*'
     - 'spec/**/*'
+Style/OptionalBooleanParameter:
+  Exclude:
+    - 'lib/**/*'
```

Comment:
It would be nice to update these at some point, but that can be a breaking change for v1.